featuretools/synthesis/get_valid_primitives.py

```diff
@@ -26,7 +26,7 @@ def get_valid_primitives(entityset, target_entity, max_depth=2, selected_primiti
 
     Args:
         entityset (EntitySet): An already initialized entityset
-        target_entity (str): Entity id of entity to create features for.
+        target_dataframe (str): Entity id of entity to create features for.
```

Comment:
Update wording in description to remove reference to entity

Reply:
I think `target_entity` here references the target entity id. Does this need to be `target_dataframe_name`?

Reply:
Yeah good catch. I think `target_dataframe_name` is appropriate here.

Reply:
updated

Reply:
Should we also update `dfs` to use `target_dataframe_name`?

Reply:
@rwedge I think it makes the expected type of input more clear if we call it `target_dataframe_name`, as the expense of a longer param name, of course...